From bec171988a7cd9f0ad8c603a2dad606990d80c80 Mon Sep 17 00:00:00 2001
From: Ken Ma <make@marvell.com>
Date: Wed, 20 Dec 2017 10:05:31 +0800
Subject: [PATCH 0087/1239] config: a8k: enable support for x3310 phy firmware
 download

In order to make X3310 PHY functional, PHY's firmware needs to be
updated. This commit patch enables support for marvell phy firmware
download in armada 8k config files.

Change-Id: Ic49f3fe051b1bf770d4317ce017b1c8b8e9d1d0d
Signed-off-by: Ken Ma <make@marvell.com>
---
 configs/mvebu_db_armada8k_defconfig   | 1 +
 configs/mvebu_mcbin-88f8040_defconfig | 1 +
 2 files changed, 2 insertions(+)

diff --git a/configs/mvebu_db_armada8k_defconfig b/configs/mvebu_db_armada8k_defconfig
index 0cc3c8240f..1652406347 100644
--- a/configs/mvebu_db_armada8k_defconfig
+++ b/configs/mvebu_db_armada8k_defconfig
@@ -36,6 +36,7 @@ CONFIG_CMD_TFTPPUT=y
 CONFIG_CMD_CACHE=y
 CONFIG_CMD_TIME=y
 CONFIG_CMD_MVEBU_BUBT=y
+CONFIG_CMD_MVEBU_PHY_FW_DOWNLOAD=y
 CONFIG_CMD_MVEBU_RX_TRAINING=y
 CONFIG_CMD_REGULATOR=y
 CONFIG_CMD_EXT4_WRITE=y
diff --git a/configs/mvebu_mcbin-88f8040_defconfig b/configs/mvebu_mcbin-88f8040_defconfig
index 032a07ae35..e6ed48b1d2 100644
--- a/configs/mvebu_mcbin-88f8040_defconfig
+++ b/configs/mvebu_mcbin-88f8040_defconfig
@@ -34,6 +34,7 @@ CONFIG_CMD_TFTPPUT=y
 CONFIG_CMD_CACHE=y
 CONFIG_CMD_TIME=y
 CONFIG_CMD_MVEBU_BUBT=y
+CONFIG_CMD_MVEBU_PHY_FW_DOWNLOAD=y
 CONFIG_CMD_REGULATOR=y
 CONFIG_CMD_EXT4_WRITE=y
 CONFIG_MAC_PARTITION=y
-- 
2.29.0

